pkg/api/validation/validation_test.go

```diff
@@ -2044,31 +2091,44 @@ func TestValidateLimitRange(t *testing.T) {
 		}
 		for i := range errs {
 			field := errs[i].(*errors.ValidationError).Field
+			detail := errs[i].(*errors.ValidationError).Detail
 			if field != "name" &&
 				field != "namespace" {
 				t.Errorf("%s: missing prefix for: %v", k, errs[i])
 			}
+			switch k {
```

Comment:
same comment as above - don't switch on the key, add the expectation to the test case struct